commit 386f6af25a4f5276765a121f98dd00d0bf6a743d
Author: David Faure <faure@kde.org>
Date:   Sun Jul 26 17:04:43 2015 +0200

    set proper cmake min req, matching kdelibs4, before project; fix build accordingly

--- CMakeLists.txt
+++ CMakeLists.txt
@@ -1,5 +1,5 @@
+cmake_minimum_required(VERSION 2.8.9)
 project(kdegtkconfig)
-cmake_minimum_required(VERSION 2.6)
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
 
 find_package(KDE4 REQUIRED)
--- gtk3proxies/CMakeLists.txt
+++ gtk3proxies/CMakeLists.txt
@@ -2,7 +2,7 @@ find_package(GTK3 REQUIRED)
 
 include_directories(${GTK3_INCLUDE_DIRS})
 
-add_definitions("-DDATA_DIR=\\\"${DATA_INSTALL_DIR}/kcm-gtk-module/\\\"")
+add_definitions(-DDATA_DIR="\\\"${DATA_INSTALL_DIR}/kcm-gtk-module/\\\"")
 
 add_executable(gtk3_preview preview3.c)
 target_link_libraries(gtk3_preview ${GTK3_LIBRARY} ${GOBJECT2_LIBRARY} ${GLIB2_LIBRARY})
--- gtkproxies/CMakeLists.txt
+++ gtkproxies/CMakeLists.txt
@@ -6,7 +6,7 @@ add_executable(reload_gtk_apps reload.c)
 target_link_libraries(reload_gtk_apps ${GTK2_LIBRARIES})
 install(TARGETS reload_gtk_apps RUNTIME DESTINATION "${KDE4_LIBEXEC_INSTALL_DIR}")
 
-add_definitions("-DDATA_DIR=\\\"${DATA_INSTALL_DIR}/kcm-gtk-module/\\\"")
+add_definitions(-DDATA_DIR="\\\"${DATA_INSTALL_DIR}/kcm-gtk-module/\\\"")
 
 add_executable(gtk_preview preview.c)
 target_link_libraries(gtk_preview ${GTK2_LIBRARIES} ${GTK2_GIOMM_LIBRARY})
